design-document.md

```diff
@@ -128,103 +132,86 @@ String email;
-  contain invalid characters: `" ' \`
-* If the employee name contains invalid characters, will throw an
-  `InvalidAttributeValueException` 
-
-![Client sends submit update employee form to Website Update/Add page. Website
-Update/Add page sends an update request to UpdateEmployeeActivity.
-UpdateEmployeeActivity saves updates to the contacts
-database.](images/UpdateEmployeeSd.png)
-
-## 6.6. _AddEmployee Endpoint_
-
-* Accepts `POST` requests to `/employee/`
-* Accepts data to create a new employee which includes their
-  firstName, lastName, jobTitle, deptId , dateOfBirth , email, and      
-  phoneNumber. Returns the new employee.
-* For security concerns, we will validate the provided employee name does not
-  contain invalid characters: `" ' \`
-* If the employee name contains invalid characters, will throw an
-  `InvalidAttributeValueException`
-
-## 6.7 _AdvancedSearch Endpoint_
-
-* Accepts the following parameters departmentId (REQUIRED), firstName,
-  lastName, and employeeID to perform a search on the database
-* Returns a list of results based on the entered search criteria.
-* A departmentID is required as this is the partition key.
-* If there is no data found, will throw a
-  `NoDataFoundException`
-* If there is no department found, will throw a
-  `InvalidDepartmentException`
-* For security concerns, we will validate the provided employee name does not
-  contain invalid characters: `" ' \`
-* If the employee name contains invalid characters, will throw an
-  `InvalidAttributeValueException`
-
-## 6.8 GetDeptGlobal 
-* Accepts `GET` requests to `/department`
-* Returns all the departments in the DepartmentTable format.
-  * If there is no data found, will throw a
-    `NoDataFoundException`
+## 6.4. _TrackExpense Endpoint_
+
+* Accepts `GET` requests to `/expense/:amount`
+* Returns the expense amount, category and note if available,
+  * If there is no expense found, will throw a
+    `InvalidExpenseException`
+
+## 6.5. _SetDueDate Endpoint_
+
+* Accepts `PUT` requests to `/expense/:date`
+* Accepts data to update a date for an expense
+* If the expense is not found, will throw a   
+  `ExpenseNotFoundException`
+
+## 6.6. _UpdateNote Endpoint_
+
+* Accepts `PUT` requests to `/expense/:note`
+* Accepts data to update the note about the expense.
+
+## 6.7 _GetSavings Endpoint_
+
+* Accepts `GET` requests to `budget/fundsAvailable
+* Returns the available funds after expenses are removed 
+  from monthly income
+
+## 6.8 _GetExpenses Endpoint_
+* Accepts `GET` requests to `/budget/:totalExpenses`
+* Returns all the categories in the Category Table format.
+    * If there are no expenses, will throw a
+      `NoExpensesFoundException` and display available funds
+
+## 6.9 _GetReminder Endpoint_
+* Accepts `GET` requests to `/expenses/:date`
+* If date of expense is after *todaysDate*, a reminder 
+  with said 'expense' will be returned.
+
+
+## 6.10 _DeleteExpense Endpoint_
+* Accepts `DELETE` request to a specific expense
```

Comment:
What is the URL for this endpoint? I would also add a description of what you'll do (i.e. what will be thrown) if the expense you're trying to delete doesn't exist.